services/spar/src/Spar/Sem/SparRoute.hs

```diff
@@ -0,0 +1,13 @@
+module Spar.Sem.SparRoute where
+
+import Data.Id (TeamId)
+import Imports
+import Polysemy
+import qualified SAML2.WebSSO.Types as SAML
+import qualified URI.ByteString as URI
+
+data SparRoute m a where
```

Comment:
note to self: fisx had a better name for this

Reply:
I forgot, but today I shall call this `SamlProtcolSettings`.